### ISSUE TITLE
SegNet: A Deep Convolutional Encoder-Decoder Architecture for Image Segmentation

### ISSUE THREAD

**Issue by guanfuchen** [guanfuchen/semseg#6]
use the code for training CamVid, here is the sample result

![image](https://user-images.githubusercontent.com/22321977/48523486-9fa74980-e8b7-11e8-8160-bac9759d91c5.png)

**Comment by guanfuchen** [comment]
Training Skills:
- We perform **local contrast normalization** [54] to the RGB input.
- We use **median frequency balancing** [14] where the weight assigned to a class in the loss function is the ratio of the median of class frequencies computed on the entire training set divided by the class frequency.
